### DISCUSSION TITLE
Tailwindcss added to Remix project breaks button event handlers

### DISCUSSION THREAD

**Discussion by rebeccao** [tailwindlabs/tailwindcss#11537] [Help]
I added Tailwindcss to a new Remix project. Tailwindcss is generating css and styles are working. However, Button events have stopped working. I've searched this issue but have not found anyone else having this problem. My package.json:
{
  "private": true,
  "sideEffects": false,
  "scripts": {
    "build": "npm run build:css && remix build",
    "build:css": "tailwindcss -m -i ./styles/app.css -o app/styles/app.css",
    "dev": "concurrently \"npm run dev:css\" \"remix dev\"",
    "dev:css": "tailwindcss -w -i ./styles/app.css -o app/styles/app.css --watch",
    "start": "remix-serve build",
    "typecheck": "tsc"
  },
  "dependencies": {
    "@remix-run/css-bundle": "^1.18.1",
    "@remix-run/node": "^1.18.1",
    "@remix-run/react": "^1.18.1",
    "@remix-run/serve": "^1.18.1",
    "isbot": "^3.6.8",
    "react": "^18.2.0",
    "react-dom": "^18.2.0"
  },
  "devDependencies": {
    "@remix-run/dev": "^1.18.1",
    "@remix-run/eslint-config": "^1.18.1",
    "@types/react": "^18.0.35",
    "@types/react-dom": "^18.0.11",
    "autoprefixer": "^10.4.14",
    "concurrently": "^8.2.0",
    "eslint": "^8.38.0",
    "postcss": "^8.4.24",
    "tailwindcss": "^3.3.2",
    "typescript": "^5.0.4"
  },
  "engines": {
    "node": ">=14.0.0"
  }
}

**Comment by rebeccao** [comment]
Here is the repo: https://github.com/rebeccao/find-lostpoets

**Comment by Sethi237** [comment]
I'm going to move this to discussions until we can identify an issue with the extension itself.

**Comment by Sethi237** [comment]
Can anyone given the resolve of mention error  https://www.marykayintouch.website/

**Comment by mattf96s** [comment]
Remix now has built in support for Tailwind. Add "tailwind: true" to remix.config.js and then remove the tailwind stuff in your package.json scripts (just the scripts section—not the actual package).

**Reply by rebeccao** [reply]
Thank you, I will add these changes!

**Reply by rebeccao** [reply]
@mattf96s I added tailwindcss to the project using your suggestion and following the instructions on the link below. Tailwind styling is working but the button events are still NOT working. Any ideas? This makes no sense.

https://tailwindcss.com/docs/guides/remix

**Reply by mattf96s** [reply]
You need to add the <Scripts /> in  your Root.tax file. This loads the JavaScript.

https://remix.run/docs/en/1.18.1/file-conventions/root

**Reply by rebeccao** [reply]
I redid the root.tsx file and this fixed the issue. Thank you @mattf96s for your patience and your help!
